### PR TITLE
Update personal.php

### LINKED ISSUE #438: header information error
## Bug report

Passman app continuously generates error in log file.

### Steps to reproduce
1. No steps here just installed app.

### Configuration
**Passman version**:    2.1.4

**Operating system**:  Ubuntu 16.04 LTS

**Web server:**. Nginx 1.12.2

**Database:** MariaDB

**PHP version:**. 7.1
      
**cloud server:** Nextcloud 

**cloud version:** 13RC3
  
 
#### Cloud log

<details>
<summary>Cloud log</summary>

Cannot modify header information - headers already sent by (output started at /var/www/nextcloud/apps/passman/personal.php:1) at /var/www/nextcloud/lib/private/AppFramework/Http/Output.php#69

</details>

No others errors appear in log file.

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/54288157-header-information-error?utm_campaign=plugin&utm_content=tracker%2F44880056&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F44880056&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by NikTh**
Closes #438

### CODE REVIEW

File: personal.php [hunk 1]
```diff
@@ -1 +1,2 @@
-<!-- Placeholder file -->
\ No newline at end of file
+<?php
+?>
```
Comment (animalillo) [inline personal.php]: Closing tags is never a good idea in php files

Comment (NikTh) [reply personal.php]: Good Point !

**Comment by worldworm** [comment]
Closes #449

**Comment by brantje** [comment]
Thank you very much for your contribution =).

**Pull request merged**